core/genesis_test.go

```diff
@@ -41,6 +41,17 @@ func TestDefaultGenesisBlock(t *testing.T) {
 	}
 }
 
+// TestDeveloperGenesisBlock checks that DeveloperGenesisBlock does not modify
+// the default value of params.AllCliqueProtocolChanges.Clique.Period.
+func TestDeveloperGenesisBlock(t *testing.T) {
+	DeveloperGenesisBlock(123, common.Address{})
+
+	period := params.AllCliqueProtocolChanges.Clique.Period
+	if period != 0 {
+		t.Errorf("Default Period changed to: %v", period)
+	}
+}
+
```

Comment:
This test is not necessary. We usually don't add tests for issues with such narrow scope.